### PR TITLE
PCT essentials

### PR DESCRIPTION
**Pull request by usr110**
and also added text for case-studies. Please view the outputs using these:

https://htmlpreview.github.io/?https://github.com/npct/pct-shiny/blob/4bfb33c1d50dc8f3f97d02fb5b4690447b3728ac/regions_www/index.html
 
https://htmlpreview.github.io/?https://github.com/npct/pct-shiny/blob/pct_essentials/regions_www/tabs/manual_body.html

P.S. The link to the pdf is specific to the branch - in this case, `pct_essentials`. Once we are happy with the changes, I will change it to the `master` branch. Please don't merge it without this change.

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Looks good to me @usr110, will await reviews from @AnnaGoodman1 and @nikolai-b as requested. Great work!

**Comment by usr110** [comment]
Cheers @Robinlovelace 

Re merge, I think it's ready. I have used a commit's SHA instead of branch name. 

@JDWoodcock have made the document a two pager.

**Comment by Robinlovelace** [comment]
One question, probably for another day/month, but would it be worth storing the source code/text of the 'essentials' document in an easy-to-edit plain text format like .md?

**Comment by usr110** [comment]
> One question, probably for another day/month, but would it be worth storing the source code/text of the 'essentials' document in an easy-to-edit plain text format like .md?

It's always easy to manage text when in plain form, so I do agree with the suggestion. We just need to think how best to keep all text at one place. Indeed something to think about for the future.

**Comment by usr110** [comment]
> 
> 
> As per my email to you, looks great but also to add/edit:
> 
> **How to use** the Propensity to Cycle Tool (PCT)
> The material here was developed **between 2016 and 2019** for use in training. To request more information or provide feedback, please email us at [pct@pct.bike](mailto:pct@pct.bike) or report issues on our GitHub issue tracker. The PCT also has a blog, which contains further information on some of the topics featured in the case studies and FAQs. Contributions to GitHub, the blog or other parts of the project are welcome.

Thanks @AnnaGoodman1 . Have made these changes. Please see the changes [here](https://htmlpreview.github.io/?https://github.com/npct/pct-shiny/blob/pct_essentials/regions_www/tabs/manual_body.html)

**Comment by Robinlovelace** [comment]
Looking good to me. While we're adding the essentials document is it worth adding links to other training resources? I've put in a PR that adds a link to the advanced training materials delivered last year and to be delivered this year: https://htmlpreview.github.io/?https://github.com/npct/pct-shiny/blob/Robinlovelace-patch-1/regions_www/tabs/manual_body.html

If @RachelAldred would you be happy to share the intermediate/intro training courses also I think the Essentials -> Introductory -> Intermediate -> Advanced links could provide good stepping stones for people to develop their understanding of the project, providing good links for every level.

File: regions_www/index.html [hunk 1]
```diff
@@ -61,7 +61,7 @@
           <div class="main-container">
             <p> Welcome to the Propensity to Cycle Tool (PCT) for England and Wales, which provides an evidence base to inform cycling investment.</p>
             <p>To run the PCT, first hover over a region on the map on the left to see the regional potential, then click on the region.</p>
-            <p>For an introduction to the tool, its use, and the scenarios, please watch the video above.</p>
+            <p>For an introduction to the tool, its use, and the scenarios, please watch the video above and read the <a target="_blank" href="https://cdn.rawgit.com/npct/pct-shiny/30b98621b37da3cb66fcf2253583af9f026f9fce/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf">Essentials</a>.</p>
```
Comment (nikolai-b) [inline regions_www/index.html]: Sorry to be a bit of a pain but I'd prefer this to be

https://npct.github.io/pct-shiny/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf

as Github pages are designed as a CDN whereas the `cdn.rawgit.com` is a bit of a hack and might change.

File: regions_www/tabs/manual_body.html [hunk 2]
```diff
@@ -14,10 +15,14 @@
...
+        <h3>How to use the Propensity to Cycle Tool (PCT)</h3>
+        <p>The material here was developed during 2016 and 2019 for use in training. To request more information or provide feedback, please email us at <a href="mailto:pct@pct.bike?Subject=Manual">pct@pct.bike</a> or report issues on our <a target="_blank" href="https://github.com/npct/pct-shiny/issues">GitHub issue tracker</a>. The PCT also has a <a target="_blank" href="https://blog.pct.bike/">blog</a>, which contains further information on some of the topics featured in the case studies and FAQs.  Contributions to GitHub, the blog or other parts of the project are welcome.</p>
 
         <div style="padding-top:6px;">
+          <h4><a name="essentials"></a>PCT Essentials</h4>
+          <ul style="line-height:2.2;">
+          <li>Read our two pager <a target="_blank" href="https://cdn.rawgit.com/npct/pct-shiny/30b98621b37da3cb66fcf2253583af9f026f9fce/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf">Essentials</a> document first, for a high level summary</li>
```
Comment (nikolai-b) [inline regions_www/tabs/manual_body.html]: same here.

Also is this meant to be:
>Read our two pager

**Comment by Robinlovelace** [comment]
> Personally I'm less happy with PDFs as a format but that's just me.

Agreed, will also help build our web presence if we have a html page with all the hyperlinks on our site. Can draft a docx -> html conversion, although I'm in favour of having both pdf and html as PDFs are a nice format for sending people and printing as a two-pager.

**Comment by Robinlovelace** [comment]
And @nikolai-b I guess this comment:

> Sorry to be a bit of a pain but I'd prefer this to be

> https://npct.github.io/pct-shiny/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf

Means we just need to wait until it's on master before the link works (as GH-pages builds from master). I agree that is consistent and makes sense.

File: regions_www/tabs/manual_body.html (hunk 2 above)
Comment (usr110) [reply regions_www/tabs/manual_body.html]: Thanks @nikolai-b 
I wanted to use it but for a test I had used raw git. Now have updated the links.

**Pull request merged**